pkgs/development/tools/analysis/radare2/default.nix

```diff
@@ -59,7 +66,11 @@ stdenv.mkDerivation rec {
   makeFlags = [
     "GITTAP=${version}"
     "RANLIB=${stdenv.cc.bintools.bintools}/bin/${stdenv.cc.bintools.targetPrefix}ranlib"
+  ] ++ lib.optionals stdenv.isDarwin [
```

Comment:
```suggestion
```

Does this break linux? If not I think we should do it like this to make linux and darwin similar.

Reply:
This would potentially also help with cross compiling.